app/controllers/admin/users/sign_ins_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Admin::Users::SignInsController < AdminController
+  layout 'admin/users'
+
+  def index
+    @title = 'Listing user sign ins'
+    @query = params[:query]
+
+    sign_ins =
+      if @query
+        User::SignIn.search(@query)
+      else
+        User::SignIn
+      end
```

Comment:
Reduce to two lines:

```suggestion
    sign_ins = User::SignIn.search(@query) if @query
    sign_ins ||= User::SignIn
```


or:

```suggestion
    sign_ins = User::SignIn
    sign_ins = sign_ins.search(@query) if @query
```

Reply:
Prefer the latter as it's clearer it's an additive filter.